src/gamatrix/__main__.py

```diff
@@ -445,9 +445,12 @@ def parse_cmdline(argv: List[str], docstr: str) -> Dict[str, Any]:
     )
     log = logging.getLogger()
 
+    version = pkg_resources.get_distribution("gamatrix").version
+
     opts = parse_cmdline(
         argv=sys.argv[1:],
         docstr=__doc__ if __doc__ is not None else "",
+        version=version,
```

Comment:
Without this new arg, `-v` was working fine but the tests failed with:

```
FAILED test/test_cmdline.py::test_new_cmdline_handling[No switches-commandline0-config_fields0-expected_values0] - NameError: name 'version' is not defined
FAILED test/test_cmdline.py::test_new_cmdline_handling[Assorted values all in one-commandline1-config_fields1-expected_values1] - NameError: name 'version' is not defined
FAILED test/test_cmdline.py::test_new_cmdline_handling[Only set the mode to server-commandline2-config_fields2-expected_values2] - NameError: name 'version' is not defined
FAILED test/test_cmdline.py::test_new_cmdline_handling[Allow the cache to update missing items.-commandline3-config_fields3-expected_values3] - NameError: name 'version' is not defined
```

Might be a better way to do it, I don't know docopt.

Reply:
Yes, this is correct - but I don't understand why it was working before. Maybe defining `VERSION` is picked up by `docopt`... will look at it later.

However, for your change, I believe this is the _correct_ way to do things. Tell `docopt` what the version is explicitly.

Reply:
Before, `VERSION` was coming in from an import, so it makes sense it was available before basically as a global var.